app/models/discourse_translator/category_translation.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module DiscourseTranslator
+  class CategoryTranslation < ActiveRecord::Base
```

Comment:
Do we need to differentiate between `Category#name` and `Category#description`? It might be useful to think about how we can expand this table to store the translation for more than a single column since some models may have multiple columns that requires us to translate.